projects/schematics/src/shared/utils/config-utils.ts

```diff
@@ -296,3 +300,60 @@ function convert(newValues: string | string[]): string {
   }
   return configValue;
 }
+
+export function getConfigs(sourceFile: SourceFile): Node[] {
+  const configs: Node[] = [];
+  const module = getModule(sourceFile);
+  if (!module) {
+    return [];
+  }
+  const literal = module.getFirstDescendantByKind(
+    tsMorph.SyntaxKind.ObjectLiteralExpression
+  );
+  if (literal) {
+    const properties = literal.getChildrenOfKind(
+      tsMorph.SyntaxKind.PropertyAssignment
+    );
+    if (properties) {
+      properties.forEach((property) => {
+        if (
+          property.getNameNode().getText() === 'providers' &&
+          property.getInitializerIfKind(
+            tsMorph.SyntaxKind.ArrayLiteralExpression
+          )
+        ) {
+          const initializer = property.getInitializerIfKind(
+            tsMorph.SyntaxKind.ArrayLiteralExpression
+          );
+          if (initializer) {
+            const elements = initializer.getElements();
+            elements.forEach((element) => {
```

Comment:
We can also streamline this to `initializer?.getElements().forEach((element) => {` and avoid having the nested `if` statement.